rhacs-operator/tasks/main.yml

```diff
@@ -0,0 +1,127 @@
+---
+- name: Testing Red Hat Advanced Cluster Security
+  block:
+    - name: Create "{{ target_namespace }}" namespace
+      kubernetes.core.k8s:
```

Comment:
You don't need this. It's done in the main tasks.

Reply:
I was still testing stuff. It's clean now.